content/ember-3-26-released.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Ember 3.26 Released
+authors:
+  - isaac-lee
+  -  # replace with real authors from the author folder (add yourself if you're not there)
+date: 2021-03-31T00:00:00.000Z
+tags:
+  - releases
+  - '2021'
+  - version-3-x
+---
+
+Today the Ember project is releasing version 3.26 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 3.27 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.26
+
+Ember.js 3.26 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 3.26 introduced 5 bug fixes. Here are a few notable ones:
+
+1. The Ember router and the `router` service have been updated so that an infinite recursion does not occur when the `router` service is injected into `app/router.js`. ([#19405](https://github.com/emberjs/ember.js/pull/19405))
+2. You can pass to an observer a dependent key whose name includes a colon. ([#19343](https://github.com/emberjs/ember.js/issues/19343))
+3. The Glimmer VM has been updated to prevent eagerly consuming arguments during modifier destruction. ([#19469](https://github.com/emberjs/ember.js/pull/19469))
+
+#### Features
+
+Ember.js 3.26 introduced 0 features.
+
+#### Deprecations
+
+Ember.js 3.26 introduced 11 deprecations.
```

Comment:
TODO:

Go through the deprecation issues and see which ones we want to bring attention to.